guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -25,38 +25,44 @@ include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[le
 endif::[]
 
 // Upgrading Satellite Server overview
-include::topics/upgrading_overview.adoc[leveloffset=+1]
+include::topics/con_upgrading-overview.adoc[leveloffset=+1]
 
-include::topics/upgrading_capsules_separately_from_satellite.adoc[leveloffset=+2]
+include::topics/con_upgrading-prerequisites.adoc[leveloffset=+1]
+
+include::topics/con_upgrade-paths.adoc[leveloffset=+1]
+
+include::topics/con_following-the-progress-of-the-upgrade.adoc[leveloffset=+1]
+
+include::topics/con_upgrading-capsules-separately-from-satellite.adoc[leveloffset=+2]
```

Comment:
Is there a reason why not to place this after Upgrading Capsule Servers?

Reply:
I think it was placed here initially because this chapter is more informative, and the user would need this information when planning the upgrade. So it makes sense to include it in the Upgrade Overview section.